src/test/java/com/selfxdsd/storage/SelfPmsITCase.java

```diff
@@ -70,4 +71,63 @@ public void returnsNullOnMissingPm() {
         MatcherAssert.assertThat(found, Matchers.nullValue());
     }
 
+    /**
+     * Picks a PM by provider name.
+     */
+    @Test
+    public void pickPm() {
```

Comment:
@criske Can you rename this to pick**s**Pm? Generally the unit tests should be verbs in first-person singural (as in "SelfPms pick**s** a PM" or "SelfPms can pick a PM"